FLAnimatedImage.android.js

```diff
@@ -1,23 +1,21 @@
 'use strict';
 
-import React, { Component } from 'react';
-
+import React, { Component} from 'react';
 import {
-  StyleSheet,
-  View,
+  Image
 } from 'react-native';
 
+import resolveAssetSource from 'react-native/Libraries/Image/resolveAssetSource';
+
+
 class FLAnimatedImage extends Component {
+
   render() {
     return (
-      <View />
+
+      <Image source={this.props.source} style={this.props.style}  />
     );
   }
 }
 
```

Comment:
What about simply export the `Image` from `react-native`?
`export { Image as default } from 'react-native'`